Robot Code/src/org/usfirst/frc/team3316/robot/commands/climbing/MoveUp.java

```diff
@@ -0,0 +1,52 @@
+package org.usfirst.frc.team3316.robot.commands.climbing;
+
+import org.usfirst.frc.team3316.robot.Robot;
+import org.usfirst.frc.team3316.robot.commands.DBugCommand;
+
+/**
+ * Command for climbing. Sets the climbing motors until we reach the target angle of the pot.
+ * 
+ * @author D-Bug
+ *
+ */
+public class MoveUp extends DBugCommand
+{
+	private double speed, initAngle;
+	private boolean onTarget;
+
+	public MoveUp()
+	{
+		requires(Robot.climbing);
+	}
+
+	protected void init()
+	{
+		initAngle = Robot.climbing.getAngle();
+	}
+
+	protected void execute()
+	{
+		speed = (double) config.get("climbing_Speed");
+
+		isFin = !Robot.climbing.setMotors(speed);
```

Comment:
The winch might draw a lot of current, especially with a low battery at the end of the match. Are we sure we want to stop the command if it overdraws?